editor/plugins/path_2d_editor_plugin.h

```diff
@@ -34,35 +34,39 @@
 #include "editor/editor_plugin.h"
 #include "scene/2d/path_2d.h"
 #include "scene/gui/box_container.h"
+#include "scene/gui/dialogs.h"
```

Comment:
This should be moved to cpp and `ConfirmationDialog` should be forward-declared instead.

Reply:
Hmm, I think all the examples I picked up declared the `ConfirmationDialog` member in the header.

I amended the commit, see if it's ok. This is the last topic to be checked in the review.